tests/setup/kfctl_openshift.yaml

```diff
@@ -47,17 +47,5 @@ spec:
         name: manifests
         path: openshift-pipelines/cluster
     name: openshift-pipelines
-  - kustomizeConfig:
-      overlays:
-        - metadata-store-mariadb
-        - ds-pipeline-ui
-        - object-store-minio
-        - default-configs
-      repoRef:
-        name: manifests
-        path: data-science-pipelines
-    name: data-science-pipelines
-  repos:
-  - name: manifests
```

Comment:
@DharmitD you're removing lines that are not part of the `kustomizeConfig` 

```
  repos:
  - name: manifests
```

Reply:
Done, added these back.